bin/test_unit

```diff
@@ -2,20 +2,17 @@
 
 set -eox pipefail
 
-rm -rf unit-test
-mkdir unit-test
-
-junit_output_file="unit-test/junit.output"
-
-rm -f "$junit_output_file"
-touch "$junit_output_file"
+junit_output_file="./junit.output"
 
 echo "Building unit test image..."
 docker build -f Dockerfile.test -t secrets-provider-for-k8s-test-runner:latest .
 
 echo "Running unit tests..."
 set +e
-  docker run --rm -t secrets-provider-for-k8s-test-runner:latest \
+  docker run --rm -t \
+             --volume "$PWD"/:/secrets-provider-for-k8s/test/ \
+             secrets-provider-for-k8s-test-runner:latest \
+             -coverprofile="./test/c.out" \
```

Comment:
is there a reason why `c.out` is the name of this file? Does this hold the coverage report? If so, I would give it a more descriptive name

Reply:
Thanks.. i agree about the c file but its came from [codeclimate](https://github.com/codeclimate/test-reporter/blob/0ad7d9c70d6b9ebca14a4394d80856391aa81a17/formatters/gocov/gocov.go)